demo/index.js

```diff
@@ -21,86 +36,56 @@ const camera = new THREE.PerspectiveCamera(
   1000
 );
 
-camera.position.set(0, 0, 125);
-
-const controls = new OrbitControls(camera, renderer.domElement);
-controls.autoRotate = true;
-controls.autoRotateSpeed = 4;
+camera.position.set(0, 50, 125);
 
 const lodeContext = lodeLoader.createContext({
   manifest,
   basePath: "./lode-build",
 });
 
-const lods = [
-  {
-    name: "assets/duck",
-    position: [30, 0, 0],
-  },
-  {
-    name: "assets/airplane",
-    position: [-30, 0, 0],
-  },
-  {
-    name: "assets/skull",
-    position: [0, 0, 30],
-  },
-  {
-    name: "assets/dragon",
-    position: [0, 0, -30],
-  },
-  {
-    name: "assets/boat",
-    position: [20, 0, -20],
-  },
-  {
-    name: "assets/armadillo",
-    position: [20, 0, 20],
-  },
-  {
-    name: "assets/apricot",
-    position: [-20, 0, -20],
-  },
-];
+const renderObjects = (scene, objs, selector = (obj) => obj) => {
+  objs.map(selector).forEach((obj, i) => {
+    for (let j = 0; j < config.objectCount; j++) {
+      const clone = obj.clone();
+      scene.add(clone);
+      clone.position.set(...generateRandomPosition(config));
```

Comment:
random positions have an effect on the used lods and therefore the performance changes "randomly"